Validation/SiTrackerPhase2V/python/Phase2OTValidateCluster_cff.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+from Validation.SiTrackerPhase2V.Phase2OTValidateCluster_cfi import Phase2OTValidateCluster
+clusterValidOT = Phase2OTValidateCluster.clone()
+
+from Configuration.ProcessModifiers.premix_stage2_cff import premix_stage2
+premix_stage2.toModify(clusterValidOT,
+    pixelSimLinkSrc = "mixData:PixelDigiSimLink",
```

Comment:
should this rather be:
```suggestion
   OuterTrackerDigiSimLinkSource = "mixData:PixelDigiSimLink",
```

based on https://github.com/cms-sw/cmssw/pull/32151/files#diff-64384dda09bca31f5fd4f195a98f471e96eda6390ba1522788568de18501ec25R452
?

Reply:
corrected the name.